src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -208,7 +119,7 @@ export class TransactionList extends Component<Props, State> {
             <View style={styles.container}>
               <View style={styles.transactionsWrap}>
                 <FlatList
-                  ListHeaderComponent={this.currentRenderBalanceBox}
+                  ListHeaderComponent={this.currentRenderBalanceBox()}
```

Comment:
what is currentRenderBalanceBox and why did it change to a function?

Reply:
Because, in my opinion, a function returns a component (JSX) whereas the property name `ListHeadComponent` grammatically is asking for a component. It may work either way but grammatically I think the change adds clarity.